### PR TITLE
cleanup-jooby-rxjava

### PR DESCRIPTION
**Pull request by feffi**
fixed minor code issues

### CODE REVIEW

File: modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java [hunk 1]
```diff
@@ -366,7 +366,7 @@ public DeferredSubscriber(final Deferred deferred) {
     @Override
     public void onCompleted() {
       if (done.compareAndSet(false, true)) {
-        deferred.resolve((Object) null);
+        deferred.resolve(null);
```
Comment (jknack) [inline modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java]: Are you sure about this? Feel we need the cast to `Object`

Comment (jknack) [reply modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java]: You're right about `resolve(Object)`, please ignore my comment.

Are you on linux? I did see that error in Travis, still can't reproduce/fix it

Comment (feffi) [reply modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java]: no problem :-) Happy to help!

Nope, I'm on macOS (so kinda posix yeah). I found this regarding the error:

https://stackoverflow.com/questions/12190377/getting-javassist-notfoundexception-with-powermock-and-powerrule-in-junit-with-m

maybe we just need to switch to the agent version? I'm not that deep into your testing (yet), but maybe this helps?

Comment (feffi) [reply modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java]: btw: locally I can't reproduce this error neither. All tests (maven/console and IntelliJ) are fine.

**Pull request merged**